examples/crewai/python/tests/test_storage.py

```
@@ -1,174 +1,256 @@
+        self.created_at = created_at
 
-        for summary in summary_results:
-            # Verify our formatting logic
-            assert "summary_type" in summary["metadata"]
-            assert "[Session Summary]" in summary["context"]  # Our formatting
 
-    def test_reset_creates_new_session_id(self):
-        """Test that reset() creates a new session with different ID."""
-        storage = HonchoStorage(user_id="test_user_reset")
+class FakeMessage:
+    def __init__(self, id, peer_id, content, metadata=None, created_at=None):
```

Ruff (0.15.13)

[error] Function argument `id` is shadowing a Python builtin

(A002)